package.json

```diff
@@ -4,7 +4,7 @@
     "node": ">=6.0.0"
   },
   "scripts": {
-    "local-docs": "npm-publish-scripts serve",
+    "local-docs": "jekyll serve --source ./docs/ --trace --config ./docs/_config.yml",
```

Comment:
Optional—this could be be a task defined in `gulp-tasks/documentation.js` that called `processPromiseWrapper('jekyll', ['serve', etc...])`

It would be nice to clear things out of `npm` `scripts`.